aws-staging-1/main.tf

```diff
@@ -200,11 +206,51 @@ module "aws_asg_org" {
   worker_docker_image_php        = "${var.latest_docker_image_garnet}"
   worker_docker_image_python     = "${var.latest_docker_image_garnet}"
   worker_docker_image_ruby       = "${var.latest_docker_image_garnet}"
-  worker_docker_self_image       = "${var.latest_docker_image_worker}"
```

Comment:
I'm having a hard time seeing why this line gets taken out?

Reply:
The main reason was to have the default staging ASG run the stable worker release, and run the latest one on the canary ASG only. The intention being to give us a more stable staging env, I'm happy to revert it though if folks think it's more valuable to always run latest, where latest means latest at time of last terraform apply.